src/frontend/next/next.config.js

```diff
@@ -18,7 +18,14 @@ if (dotenv.config({ path: envPath }).error) {
   }
 }
 
-module.exports = {
+// Required by about.mdx, along with the module.exports
+const withMDX = require('@next/mdx')({
+  extension: /\.mdx?$/,
+});
+
+module.exports = withMDX({
+  pageExtensions: ['ts', 'tsx', 'md', 'mdx'],
   poweredByHeader: false,
   reactStrictMode: true,
-};
+  distDir: 'build',
```

Comment:
Does our Netlify deploy not expect us to put things in `out/`? https://github.com/Seneca-CDOT/telescope/blob/master/netlify.toml#L3

cc @raygervais for tips

Reply:
That's correct, we had configured Netlify to use /out I thought for NextJS. Changing this would require us also changing the output path in Netlify

Reply:
@chrispinkney let's stick with `out` here to match our deploy.

Reply:
Hey yall, 

Sorry for the mix up guys. I just based it `distDir: 'build'` off the [history of the file](https://github.com/Seneca-CDOT/telescope/commit/bbda77b488bfe8768517b492c666eea6727fb93b#diff-c3877fbd3e2c4a17fe86388d235278d2eb5c5c367d71c9a147463d2c2b7c6706), **but just to confirm this needs to be `out`, right?**

Reply:
Agree, let's solve this: @raygervais can you untangle what's going on here for me?  build vs. out vs. .next? https://github.com/Seneca-CDOT/telescope/commit/bbda77b488bfe8768517b492c666eea6727fb93b#diff-ab8f79b68b7adff7a07db953bf453f3c5aa6ade98d2b1b67d8432b36392489ed

Reply:
According to the configuration here and in our repo, it should be: `out`.  It appears I need to update the docs for this.

Also @humphd, when you have a moment let me get you setup on Telescope + Netlify so that I'm not the blocker in the case which I am not available at the time.

![image](https://user-images.githubusercontent.com/14265337/105392471-3e8f0500-5be9-11eb-826b-ea48d1aae709.png)